components/channel_view/channel_view.jsx

```diff
@@ -11,7 +11,7 @@ import deferComponentRender from 'components/deferComponentRender';
 import ChannelHeader from 'components/channel_header';
 import CreatePost from 'components/create_post';
 import FileUploadOverlay from 'components/file_upload_overlay.jsx';
-import PostView from 'components/post_view';
+import PostList from 'components/post_view/post_list';
```

Comment:
Just change this to PostList - similar to component name - to make it more searchable.